hydra-node/src/Hydra/Chain/Direct/Wallet.hs

```diff
@@ -600,13 +596,8 @@ stateQueryClient tracer tipVar utxoVar address =
 -- Keys
 --
 
-generateKeyPair :: IO (VerificationKey, SigningKey)
-generateKeyPair = do
-  ( Cardano.Api.PaymentVerificationKey (Ledger.VKey vk)
-    , Cardano.Api.PaymentSigningKey sk
-    ) <-
-    generate genKeyPair
-  pure (vk, sk)
+generateKeyPair :: IO (VerificationKey PaymentKey, SigningKey PaymentKey)
+generateKeyPair = generate genKeyPair
```

Comment:
Feel almost useless now :upside_down_face:

Reply:
It probably alleviates the need for some imports or dependencies on ledger's test packages. I will try to inline it and see where this goes.